src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100) + 1;
```

Comment:
same for this Random and ColorSupplier
100 should be class constant, don't forget to use CONSTANT_CASE